qa/rpc-tests/test_framework/util.py

```diff
@@ -20,6 +20,12 @@
 from authproxy import AuthServiceProxy, JSONRPCException
 from util import *
 
+def null_file():
+    if os.name == 'nt':
+        return open("NUL", "w+")
+    else:
+	return open("/dev/null", "w+")
+
```

Comment:
The devnull switch is not required.
I think you can do `devnull = open(os.devnull, 'w')` at L96 and L188.